src/licenses/agplv3.ts

```diff
@@ -658,8 +658,9 @@ to attach them to the start of each source file to most effectively
 state the exclusion of warranty; and each file should have at least
 the "copyright" line and a pointer to where the full notice is found.
 
-    <one line to give the program"s name and a brief idea of what it does.>
-    Copyright (C) <year>  <name of author>
+    <one line to give the program's name and a brief idea of what it does.>
+
+    Copyright (C) {{ year }}  {{ organization }}
```

Comment:
It seems the template is `Copyright (c) <year> <name of author>` according to the original file.
https://www.gnu.org/licenses/agpl-3.0.txt

Reply:
Indeed, fixed in 1c6c5ec91eb0a59ee134bb866f459d9fcad0f1ef